mesos-simulation/src/main/scala/mesosphere/mesos/simulation/DriverActor.scala

```diff
@@ -45,9 +48,21 @@ class DriverActor(schedulerProps: Props) extends Actor {
 
   private[this] val numberOfOffersPerCycle: Int = 10
 
+  private val randomSeed: Long = 123
+  // use seed for getting more deterministic behavior
+  private[this] val random = new Random(new java.util.Random(randomSeed))
```

Comment:
I have a dejavu. Didn't I answer this yet?

Well, we have an integration test that has a timeout. There are some random scenarios where this will fail so I wanted to keep this fixed for now. 

We can change this later or make this configurable to have the tests testing more scenarios. But this is for sure better than what we did before. 

Does that make sense to you?